src/System.IO.FileSystem/src/System/IO/UnixFileSystem.cs

```diff
@@ -37,10 +34,10 @@ public override void CopyFile(string sourceFullPath, string destFullPath, bool o
             // Copy the contents of the file from the source to the destination, creating the destination in the process
             const int bufferSize = FileStream.DefaultBufferSize;
             const bool useAsync = false;
-            using (Stream src = new FileStream(sourceFullPath, FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize, useAsync))
-            using (Stream dst = new FileStream(destFullPath, overwrite ? FileMode.Create : FileMode.CreateNew, FileAccess.ReadWrite, FileShare.None, bufferSize, useAsync))
+            using (var src = new FileStream(sourceFullPath, FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize, useAsync))
+            using (var dst = new FileStream(destFullPath, overwrite ? FileMode.Create : FileMode.CreateNew, FileAccess.ReadWrite, FileShare.None, bufferSize, useAsync))
```

Comment:
Could the handles be opened directly or use `UnixFileStream` instead of `FileStream` to avoid the extra wrapper Stream allocations?